src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -89,3 +90,63 @@
                                                 {:event  :wallet/get-suggested-routes
                                                  :error  error
                                                  :params request-params}))}]})))
+
+(rf/reg-event-fx :wallet/add-authorized-transaction
+ (fn [{:keys [db]} [transaction]]
+   (let [transaction-hashes (:hashes transaction)
+         chain-id           (key (first transaction-hashes))
+         tx-id              (first (val (first transaction-hashes)))
```

Comment:
Why do we only get the first item? what is contained in the rest of them?

Reply:
ah this is a temporary solution, will adjust properly when we go to implement the transaction progress page. 
Happy to remove it for now, I think I left it there to also open up discussion about how we store transactions in the db

Reply:
Discussed with @ulisesmac - will create an issue for this feature and mention this needs to be reworked. 👍

Reply:
https://github.com/status-im/status-mobile/issues/18307